tools/enable_local_firecracker.sh

```diff
@@ -79,5 +79,9 @@ if [ "$FOUND_ENTRY" != "$IP_ENTRY" ]; then
     echo "$IP_ENTRY" | EDITOR='tee -a' visudo >/dev/null
 fi
 
+# Clean up vestigial network namespaces, these can be left around if firecracker
+# doesn't exit cleanly and interfere with subsequent VM startup.
+ip --all netns delete
```

Comment:
How would you feel about running this when the executor binary started up instead?

We sort of already assume that only one executor is running per-machine in prod, so this would be safe, and then it would also happen more regularly (since people almost never use this script)

Reply:
Sounds good, done.